src/redux/reducers/auth.js

```diff
@@ -0,0 +1,23 @@
+import { LOGIN_ERROR, LOGIN_SUCCESS } from '../actions/actionsTypes';
+
+const initialState = {
+  user: null,
+  error: null,
+};
+
+const authReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case LOGIN_SUCCESS:
+      return {
+        ...state, user: action.payload,
+      };
+    case LOGIN_ERROR:
+      return {
+        ...state, error: action.payload,
+      };
+    default:
+      return state;
+  }
+};
```

Comment:
There's an alternative way of not using switch statements, see if you can find it.